### DISCUSSION TITLE
Minor suggeston for Italian translations

### DISCUSSION THREAD

**Discussion by kebekus** [Akaflieg-Freiburg/enrouteText#40] [Team Posts]
Dear all,

I have been contacted by Antonio Gaiano, who is currently the CEO of Flybox Avionics Italy. He was very enthusiastic about the "Enroute Flight Navigation", but had one minor suggestion for improvement of the Italian translation:

```
Also, I don't want to overwhelm whoever did the Italian translation, 
but instead of "Itinerario" in the main menu, it would be much better 
to put "Rotta", it's more aeronautical...
```

Perhaps you could consider that?

Best wishes -- and thanks to you all,

Stefan.

**Comment by TTC70** [comment]
Hello Stefan,
Regarding that specific suggestion, for me it's a negative.
"Itinerary" (itinerario, as it is now) gives a better idea of what you can
see at that menu, that is the "itinerary" as set of singles routes which
form the final "path" of the flight an user is planning.
"Route" ("rotta") are the settings of the single leg (heading, speed,
altitude, and so on).

Personally, when I press "rotta" (route) I expect mainly the heading... But
the heading of what?
While when I press "itinerario" (itinerary) I see the data related to all
the legs I planned.

For me, for the main menu, "itinerario" Is the best choice. I'll give to
Ivan the last word though.

Have a nice Sunday
Antonio




Il Dom 14 Nov 2021, 11:01 Stefan Kebekus ***@***.***> ha
scritto:

**Comment by Ivan081** [comment]
Rotta for me is only for one direction, Itinerario is for a path. The best
but long Is 'Piano di volo' (Flight Plan)

Il dom 14 nov 2021, 11:01 Stefan Kebekus ***@***.***> ha
scritto:

**Comment by TTC70** [comment]
Stefan,
I agree with "Piano di volo".
Regarding the length, the following voice of the menu is "Destinazioni"
which is about the same length if compared to "Piano di volo".

I have the file for the italian translation opened at this moment. I am
going to make that modification.
Let me take the advantage of this email asking you to "free" the item
"Traffic receiver", I found a typo in the translation "Connected to traffic
receiver at IP address 192.168.1.1, port 2000. Receiving traffic
information …", at the moment it is "Collegato al ricevitore traffico
all'indirizzo IP 192.168.1.1, porta 2000. Ricezzione informazioni
traffico..."... if you want/can please replace the word "Ricezzione" with
the word "Ricezione".

Best regards
Antonio

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Mail
priva di virus. www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

Il giorno dom 14 nov 2021 alle ore 14:04 Ivan ***@***.***> ha
scritto:

**Comment by TTC70** [comment]
Ok, done.
Please let me know if you want to replace "Itinerario" with "Piano di volo"
everywhere.... At the moment I did it only for the main menu.

Cheers
Antonio

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Mail
priva di virus. www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

Il giorno dom 14 nov 2021 alle ore 14:09 Antonio Fardella <
***@***.***> ha scritto:

**Comment by kebekus** [comment]
Am 14.11.21 um 14:10 schrieb TTC70:

> I agree with "Piano di volo".
>
> Regarding the length, the following voice of the menu is "Destinazioni"
>
> which is about the same length if compared to "Piano di volo".
>
Very well then!

> I have the file for the italian translation opened at this moment. I am
>
> going to make that modification.
>
Thanks.

> Let me take the advantage of this email asking you to "free" the item
>
> "Traffic receiver", I found a typo in the translation "Connected to 
> traffic
>
> receiver at IP address 192.168.1.1, port 2000. Receiving traffic
>
> information …", at the moment it is "Collegato al ricevitore traffico
>
> all'indirizzo IP 192.168.1.1, porta 2000. Ricezzione informazioni
>
> traffico..."... if you want/can please replace the word "Ricezzione" with
>
> the word "Ricezione".
>
Sure. Go ahead.

Best wishes from the foggy Black Forest,

Stefan.

-- 
Albert-Ludwigs-Universität Freiburg
Mathematisches Institut
Ernst-Zermelo-Str. 1
79104 Freiburg / Germany

Phone: +49 761 203-5536

Web:             https://cplx.vm.uni-freiburg.de
Federated-Cloud: ***@***.***
Matrix:          @sk380:mtrx.mathematik.uni-freiburg.de

**Comment by TTC70** [comment]
Last thing.... i don't have access to some lines... they appear to be grayed out and i cannot write...

**Comment by TTC70** [comment]
Ok..... done.... I replaced all "Itinerario" with "Piano di volo. I cannot have the access to some of the lines because those lines are grayed our.

**Comment by kebekus** [comment]
Oops. It seems that I never replied to this comment. I apologize.

The translation items in gray are outdated items that are no longer used by the app, but remain in place so they can be re-used if the same (or a similar) text shows up in the future. I remove them from time to time. Meanwhile, you can just ignore them

Best,

Stefan
